vault/resource_gcp_auth_backend.go

```diff
@@ -278,21 +306,32 @@ func gcpAuthBackendRead(d *schema.ResourceData, meta interface{}) error {
 			return err
 		}
 	}
+
 	// fetch AuthMount in order to set accessor attribute
-	mount, err := getAuthMountIfPresent(client, d.Id())
+	mount, err := getAuthMountIfPresent(client, gcpPath)
 	if err != nil {
 		return err
 	}
 	if mount == nil {
 		d.SetId("")
 		return nil
 	}
-	if err := d.Set("accessor", mount.Accessor); err != nil {
+	log.Printf("[DEBUG] Reading %s auth tune from '%q/tune'", gcpAuthType, gcpAuthPath)
+	rawTune, err := authMountTuneGet(client, gcpAuthPath)
+	if err != nil {
+		return fmt.Errorf("error reading tune information from Vault: %w", err)
+	}
+	data := map[string]interface{}{}
+	data["tune"] = []map[string]interface{}{rawTune}
+	if err := util.SetResourceData(d, data); err != nil {
 		return err
 	}
 
+	if err := d.Set("accessor", mount.Accessor); err != nil {
```

Comment:
Looks like this was missing, but we should also add a similar block to set the description since it is available in the mount :)

Reply:
Thanks for catching that:  Added a block for `description` and also added resource attribute checks for the `description` in the tests.